packages/components/pie-switch/test/component/pie-switch.spec.ts

```diff
@@ -28,6 +28,17 @@ test.describe('Component: `Pie switch`', () => {
         await expect(pieSwitch).toBeVisible();
     });
 
+    test('should have a "visible" input to help with accessibility', async ({ mount, page }) => {
```

Comment:
Is this to account for the off screen input i.e the way we hide it? If so could the description be updated to something like below?

`should have an input element that is a readable and appropriately associated input element for screen readers.`

Reply:
Previously we were hiding it with `appearance: none;` which NVDA didn't like, because it didn't want to describe an input which wasn't shown. Now we just make it only a single pixel in size; this way it's still technically displayed and the screen reader likes it.

Reply:
Yeah so i think the test description might read better to explain that. More of a NIT. At first i thought the input was visible on screen but saw the style changes and realised it was hidden with the css using the 1px method.